docs/initialization.md

```diff
@@ -50,10 +50,10 @@ ebcli tx ethbridge create-claim --help
 # Now you can test out the ethbridge module by submitting a claim for an ethereum prophecy
 # Create a bridge lock claim (Ethereum prophecies are stored on the blockchain with an identifier created by concatenating the nonce and sender address)
 # ebcli tx ethbridge create-claim [ethereum-chain-id] [bridge-contract] [nonce] [symbol] [token-contract] [ethereum-sender-address] [cosmos-receiver-address] [validator-address] [amount]",
-ebcli tx ethbridge create-claim 3 0xC4cE93a5699c68241fc2fB503Fb0f21724A624BB 0 eth 0x0000000000000000000000000000000000000000 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 $(ebcli keys show testuser -a) $(ebcli keys show validator -a --bech val) 3eth lock --from=validator --chain-id=peggy --yes
+ebcli tx ethbridge create-claim 0xC4cE93a5699c68241fc2fB503Fb0f21724A624BB 0 eth 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 $(ebcli keys show testuser -a) $(ebcli keys show validator -a --bech val) 3eth lock --token-contract-address=0x0000000000000000000000000000000000000000 --ethereum-chain-id=3 --from=validator --chain-id=peggy --yes
 
 # Then read the prophecy to confirm it was created with the claim added
```

Comment:
we should add the output of `--help` here too actually

Reply:
https://github.com/cosmos/peggy/pull/175/commits/a1bed706b60a52d3c88b8e3be332e1c0a1979959, https://github.com/cosmos/peggy/pull/175/commits/afdffa5a7bd17853d6f81f38b486e610ab3f36a2

Reply:
`bridge-registry-contract`

Reply:
see https://github.com/cosmos/peggy/pull/175#discussion_r402083717